ontopy/ontology.py

```diff
@@ -283,39 +283,45 @@ def get_by_label(
         The current implementation also supports "*" as a wildcard
         matching any number of characters. This may change in the future.
         """
-        # pylint: disable=too-many-arguments,too-many-branches
+        # pylint: disable=too-many-arguments,too-many-branches,invalid-name
```

Comment:
Please use 'valid names' instead.

Reply:
My point was not to introduce pylint exceptions.

Reply:
But I thought that we all agree that s, p, o are valid names in triples.

Reply:
OK, but not for the a in lines 330 to 332

Reply:
OK, I added some extra characters to those `a`'s. But I really don't think it improves anything - just makes the lines longer...